src/components/application_manager/rpc_plugins/rc_rpc_plugin/test/commands/rc_get_interior_vehicle_data_consent_test.cc

```diff
@@ -0,0 +1,238 @@
+        .WillByDefault(Return(application_manager::HmiInterfaces::
+                                  InterfaceState::STATE_AVAILABLE));
+    ON_CALL(app_mngr_, application(kAppId)).WillByDefault(Return(mock_app_));
+    ON_CALL(mock_allocation_manager_, GetApplicationExtention(_))
+        .WillByDefault(Return(rc_app_extention_));
+    ON_CALL(app_mngr_, GetPolicyHandler())
+        .WillByDefault(ReturnRef(mock_policy_handler_));
+    ON_CALL(app_mngr_, hmi_capabilities())
+        .WillByDefault(ReturnRef(mock_hmi_capabilities_));
+    ON_CALL(mock_hmi_capabilities_, rc_capability())
+        .WillByDefault(Return(&rc_capabilities_));
+    ON_CALL(mock_policy_handler_,
+            CheckHMIType(
+                _, mobile_apis::AppHMIType::eType::REMOTE_CONTROL, nullptr))
+        .WillByDefault(Return(true));
+    ON_CALL(mock_allocation_manager_, is_rc_enabled())
+        .WillByDefault(Return(true));
+  }
+
+  template <class Command>
+  application_manager::SharedPtr<Command> CreateRCCommand(
+      MessageSharedPtr& msg) {
+    InitCommand(kDefaultTimeout_);
+    return ::utils::MakeShared<Command>(msg ? msg : msg = CreateMessage(),
+                                        app_mngr_,
+                                        rpc_service,
+                                        mock_hmi_capabilities_,
+                                        mock_policy_handler_,
+                                        mock_allocation_manager_);
+  }
+
+  MessageSharedPtr CreateBasicMessage() {
+    MessageSharedPtr message = CreateMessage();
+    (*message)[application_manager::strings::params]
+              [application_manager::strings::function_id] =
+                  mobile_apis::FunctionID::ButtonPressID;
+    (*message)[application_manager::strings::params]
+              [application_manager::strings::connection_key] = kConnectionKey;
+    (*message)[application_manager::strings::params]
+              [application_manager::strings::connection_key] = kAppId;
+    return message;
+  }
+
+ protected:
+  utils::SharedPtr<MockApplication> mock_app_;
+  MockRequestControlerSettings mock_request_controler;
+  MockProtocolHandler mock_protocol_handler;
+  MockHMIMessageHandler mock_hmi_handler;
+  am::CommandHolderImpl command_holder;
+  testing::NiceMock<rc_rpc_plugin_test::MockResourceAllocationManager>
+      mock_allocation_manager_;
+  smart_objects::SmartObject rc_capabilities_;
+  MockRPCPlugin mock_rpc_plugin;
+  MockCommandFactory mock_command_factory;
+  am::request_controller::RequestController request_controller;
+  am::rpc_service::RPCServiceImpl rpc_service;
+  utils::SharedPtr<RCAppExtension> rc_app_extention_;
+  utils::SharedPtr<am::plugin_manager::MockRPCPluginManager>
+      mock_rpc_plugin_manager;
+};
+
+TEST_F(RCGetInteriorVehicleDataConsentTest,
+       Run_MobileSendButtonPressMessage_HMISendASKDRIVERModeToMobile) {
+  MessageSharedPtr mobile_message = CreateBasicMessage();
```

Comment:
@ValeriiMalkov please keep paradigm AAA (http://wiki.c2.com/?ArrangeActAssert) And, split this blocks with empty line

Reply:
@LuxoftAKutsan fixed at the commit 051264163471fb3bb21b58ace83bbb7f9cf89b84